react-accordion-component/src/Accordion.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react';
+
+export default function Accordion({ title, content }) {
```

Comment:
This is a nice, simple solution. But...

Since each Accordian keeps track of its own "open" state, there is no way for the Accordian to close when you open a second one. You need to "pull the state up" to the container and let it determine which Accordian should be open.